artifacts/backup-cronjob.yaml

```diff
@@ -26,23 +26,19 @@ spec:
           containers:
           - name: backup
             image: grtl/xtrabackup:latest
-            env:
-            - name: JOB_NAME
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.name
             command:
             - bash
             - "-c"
             - |
               set -ex
 
               # Clone data from master.
-              mkdir /mysql/backup/{{ .Name }}/${JOB_NAME}
-              ncat --recv-only {{ .Spec.Cluster }}-$((0)).{{ .Spec.Cluster }} 3307 | xbstream -x -C /mysql/backup/{{ .Name }}/${JOB_NAME}
+              DATE=`date '+%Y-%m-%d-%H-%M'`
+              mkdir /mysql/backup/{{ .Name }}/${DATE}
+              ncat --recv-only {{ .Spec.Cluster }}-$((0)).{{ .Spec.Cluster }} 3307 | xbstream -x -C /mysql/backup/{{ .Name }}/${DATE}
```

Comment:
`{{ .Spec.Cluster }}-$((0))` is the same as `{{ .Spec.Cluster }}-0`

Reply:
Done